python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -379,6 +379,13 @@ def get_group(self, name, obj=None):
         """
         if obj is None:
             obj = self.obj
+        else:
+            warnings.warn(
+                "obj is deprecated and will be removed in a future version. "
+                "Do ``df.iloc[gb.indices.get(name)]`` "
```

Comment:
We will have to implement indices property in groupby class no? Or is it already implemented?

Reply:
Good point, I thought that was an attribute that we already exposed but apparently not. I think the functionality should exist for this in libcudf but I'm not sure if we ever extract it in Python. Do you know? Do you think we should wait to merge this PR until after we add the feature? I'm not sure how much work it would be, I can scope at least.

Reply:
I opened a PR that implements `indices` in python here: https://github.com/rapidsai/cudf/pull/14912

Reply:
Thanks.